### PR TITLE
Added basic logger

### PR DESCRIPTION
**Pull request by oosidat**
- Also initialized linting config, package.json, readme

### CODE REVIEW

**Comment by oosidat** [comment]
Not sure how to assign and/or publish to npm @dougmoscrop

File: index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const bunyan = require('bunyan'),
+	streams = require('service-logging-streams');
+
+module.exports = function(options) {
+	return bunyan.createLogger({
```
Comment (dougmoscrop) [inline index.js:8]: I would prefer that we require a name -- that removes the two || else conditions on filename and name.

**Comment by dougmoscrop** [comment]
I will publish after merge

**Comment by oosidat** [comment]
Now throwing error when `options.name` is falsy

**Pull request merged**